src/Gui/SoDatumLabel.cpp

```diff
@@ -519,26 +521,33 @@ void SoDatumLabel::computeBBox(SoAction * action, SbBox3f &box, SbVec3f &center)
 SbVec3f SoDatumLabel::getLabelTextCenter()
 {
     // Get the points stored
+    int numPts = this->pnts.getNum();
+    if (numPts < 2) {
```

Comment:
The ANGLE case currently use only 1 point the second is undefined. It can be a copy/paste issue at
https://github.com/FreeCAD/FreeCAD/blob/fa8f29aed489247e931631dbd1e600cc7610c38f/src/Mod/Sketcher/Gui/EditModeConstraintCoinManager.cpp#L1449-L1454

Nevertheless the 1 point case shouldn't return here.

Reply:
@wwmayer is this check for < 2 correct?

Reply:
Yes, it's correct. This function requires two points and usually when the node is used in Sketcher this is the case. But this is a security check to make sure no undefined behaviour can occur.